provision.sh

```diff
@@ -118,7 +120,21 @@ set -o errexit
 
 
 sudo apt-get update
-sudo apt-get install -y netcat openssl libpcre3 dnsmasq procps perl
+sudo apt-get install -y netcat libpcre3 dnsmasq procps perl
+
+if [ semverEQ "$KONG_VERSION" "0.14.0" -o semverGT "$KONG_VERSION" "0.14.0" ]; then
+  wget https://www.openssl.org/source/openssl-1.1.0h.tar.gz
+  tar xzvf openssl-1.1.0h.tar.gz
+  cd openssl-1.1.0h
+  ./config -Wl,--enable-new-dtags,-rpath,'$(LIBRPATH)'
+  make
+  sudo make install
+  cd ..
+else
+  sudo apt-get install -y openssl
+fi
```

Comment:
Is installing openssl here even necessary? The dist packages seem to be bundle it already (as of today, it even happens so that both Nginx and luaossl are statically linked against libssl and libcrypto). I wonder why openssl/libpcre/dnsmasq and perl were added here?

Reply:
good point

Reply:
@Tieske Mind giving a try to removing those dependencies and see if it would work? I do not have the proper setup for Vagrant anymore...

Reply:
@thibaultcha already testing. Just pushed a commit to master. Only `dnsmasq` has to remain for Kong 0.9.x.